src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -255,11 +255,18 @@ export default App.extend({
       },
       'delete'() {
         const itemCount = this.selected.length;
-        this.selected.destroy().then(() => {
-          this.showDeleteSuccess(itemCount);
-          app.stop();
-          this.getState().clearSelected();
-        });
+
+        this.selected.destroy({ wait: true })
```

Comment:
I think we should revert this, so it's clear _something_ needs to be done.. I don't think this'd what it'd be if we do this.